py3status/modules/net_iplist.py

```diff
@@ -77,7 +77,7 @@ class Py3status:
 
     def post_config_hook(self):
         self.iface_re = re.compile(r"\d+: (?P<iface>[\w\-]+):")
-        self.ip_re = re.compile(r"\s+inet (?P<ip4>[\d\.]+)(?:/| )")
+        self.ip_re = re.compile(r"\s+inet (?P<ip4>[\d.]+)(?:/| )")
```

Comment:
ain't that catch "any caracter" instead of the intended "dot" ?

Reply:
No.
https://docs.python.org/3/library/re.html#regular-expression-syntax

    Special characters lose their special meaning inside sets. For example, `[(+*)]` will match any of the literal characters `'('`, `'+'`, `'*'`, or `')'`.

Reply:
No.
https://docs.python.org/3/library/re.html#regular-expression-syntax

    Special characters lose their special meaning inside sets. For example, `[(+*)]` will match any of the literal characters `'('`, `'+'`, `'*'`, or `')'`.

Reply:
wow TIL something new again <3